Portfolio: setup and mobile first

shurichmond opened the pull request:
In this Pull Request, I have done the following

- [x] Setup my project based on the Figma template I choose 
- [x] Included Linters in the project so as to follow best practices 
- [x] Included a descriptive README file 

The following task related to the project (Portfolio) were handled in this Pull Request 

- [x] Included the toolbar (or header)
- [x] Included the headline section (right after the header

shurichmond committed:
Created my index and CSS file

shurichmond committed:
Added .ignore file

shurichmond committed:
Added linters configuration

shurichmond committed:
Added jason files

shurichmond committed:
Installed stylelint and Added stylelintrc file

shurichmond committed:
Fixed Linter Errors

shurichmond committed:
Updated My README file

Lizdev-05 commented on README.md:
```diff
@@ -1,2 +1,66 @@
+    <li><a href="https://github.com/microverseinc/linters-config">Linters</a></li>
+  </ul>
+</details>
+
```
Great job providing a README file for your users 👍. Kindly update this README file with the new template provided by Microverse, you can access it here 👉 [README](https://github.com/microverseinc/readme-template)☺️⚔️

Lizdev-05 commented on index.html:
```diff
@@ -0,0 +1,58 @@
+<body>
+
+        <div class="container">
+
+            <section class="Header">
+                
+                    <ul class="header-logo">
+                        <li class="logo"> My Logo</li>
+                        <li class="hamburg-logo"> <img src="Img/Hlogo.png" alt=""></li>
+                
+                    </ul>
```
Great job implementing the nav👍. Kindly make use of `header` tag and `nav` tag for this section, this is in accordance to HTML semantic tag usage.
```
`<header>
   <a href="#"> LOGO</a>
    <nav>
    <img src="Img/Hlogo.png" alt="">
   </nav>
 </header>`
```

Kindly note that your logo should be wrapped in an `a` tag so that users can navigate back to the home page by clicking on the logo. 😎

Lizdev-05 commented on index.html:
```diff
@@ -0,0 +1,58 @@
+                
+                    </ul>
+
+                    <div class="section-header">
+
```
Kindly make use of the `section` tag instead of the `div` tag in accordance with HTML semantic tag usage. You can read more about it [here](https://www.freecodecamp.org/news/semantic-html5-elements/) 😊

Lizdev-05 reviewed (changes requested):
Hi @shurichmond, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<img src="https://media0.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif"/>
## Highlights
- Correct GitHub Flow 👍

- Good commit history 💰

- Good PR title/description 💯

- All linters passed 🏅

## Required Changes ♻️

- [ ] Kindly make add the background images to your project in line with the Figma design.
- [ ] Please, take note of the color, fonts, images, etc.

**Your Design**
<img width="415" alt="image" src="https://user-images.githubusercontent.com/73854999/205438027-b08914de-b13e-43e7-9a80-c87f6e3fd6ec.png">

**Figma Design**
<img width="415" alt="image" src="https://user-images.githubusercontent.com/73854999/205438084-7917c322-8c42-45b0-a34b-ae08dd9f2030.png">


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

shurichmond committed:
Asigned a font color to Primary text

shurichmond committed:
Fixed background image reference

shurichmond committed:
Fixed Stylint Error

amejid reviewed (commented):
Hi @shurichmond, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [x] You have submitted a project, that does not have a meaningful Pull Request title and description, and a README file following the [template](https://github.com/microverseinc/readme-template) provided by Microverse. Kindly use the correct README template. The only optional section is the `FAQ` section.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

shurichmond commented:
Pease My Pull Request Description was approved by the first Reviewer.  I used the Template for my README which was also approved. So please I don't know why I'm getting these reviews again. I don't know what to do now.  I had to work on my background Image which I did. Please can you review

huseyinbiyik commented:
Hi @shurichmond,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

![Alt Text](https://media.giphy.com/media/xUPGcwE3CUkXlvonug/giphy.gif)

## Invalid Code Review Request

- Please notice your Readme file was approved last week, not this week. Kindly make sure all the sections are present in your [Readme](https://github.com/microverseinc/readme-template) file. Just the FAQ section is optional. Please also remember to add the MIT.md license file and replace all the placeholders with your project info. Please do it once, and you can use it again for the next projects which will save time. ⏳

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

shurichmond committed:
Updated README file

shurichmond committed:
Added files to gitignore

shurichmond committed:
Deleted Unecceary files

shurichmond commented:
Worked on My README file as required

- [x]  All sections added to README as In Template
- [x] Added MIT.md License file and Place Holders replaced

BregornOriginal commented on index.html:
```diff
@@ -0,0 +1,59 @@
+</head>
+<body>
+
+        <div class="container">
```
- [x] Kindly change this div tag for a section tag. To follow HTML semantic tag usage 💯

BregornOriginal commented on index.html:
```diff
@@ -0,0 +1,59 @@
+
+        <div class="container">
+
+            <section class="Header">
```
- [x] This `section` tag should be a `header` tag. Since inside your section on this part of the proyect you will have a final structure like this:
````
<section>
  <header>
    <nav>
      <a> your logo </a>
      <ul>
        <li> hamburger icon </li>
      </ul>
    </nav>
  </header>
  <div>
  <h1> here are the tag elements for your name </h1>
  <p> next a `p` for your presentation </p>
  </div>
  <div>
    <a> for your let's connect </a>
    <ul> for your list of social icons
      <li> your list of social icons </li>
    </ul>
  </div>
</section>
````

***This is the same approach that the previous reviewer mentioned with a little improvement, so you can see it easily***

BregornOriginal commented on css/styles.css:
```diff
@@ -0,0 +1,74 @@
+  padding-right: 1.5rem;
+}
+
+.section-header {
+  background-image: url(../Img/Header-img.png);
+  background-repeat: no-repeat;
+  background-clip: content-box;
+  background-size: cover;
+  background-position: 80% 5%;
+}
```
Excelent work adding your background image on the section 💯 💯 💯 
After fixing the different elements that you have to fix, kindly fix the background image. Since this one has to cover all the height of the view.
- [x] To do this, you can use a `height` 100vh on your section that includes the background image, in this way the height of your image will cover the view screen 💯 💯 💯 

![image](https://user-images.githubusercontent.com/93630700/206809176-68dd98de-4763-4438-9787-3dfc64c12095.png)

BregornOriginal reviewed (changes requested):
## Hi @shurichmond  

## Good job so far!
## There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

✅ Linters are passing
✅ Good commit history
✅ Excelent Pull Request description
✅ Your README file look professional now 🥇 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

shurichmond committed:
Changed HTML structure to follow HTML semantic tag usage

shurichmond commented:
I have changed the structure of the project as Required. 

- [x] Changed div tag for a section tag.
- [x]  This section tag should be a header
- [x]  Fixed the background image

shurichmond committed:
Modified css styles (fixing stylint)

Nyame-Wolf commented on index.html:
```diff
@@ -0,0 +1,49 @@
+
+<body>
+
+    <section class="container">
```
As requested by the previous reviewer, kindly remove this section tag here. The structure of your code should be:
`<html>`
`<body>`
inside body:
      `<header>`
        `<main>`
          `<footer>`

The `header` tag does not need to be in a section tag. Just have it as a `header`
It is important to follow hierarchy and use the correct semantic tags so as not to confuse screen readers.

shurichmond replied:
The Previous reviewer said I should use a Section tag as shown above. You are saying I should remove the section tag. I am repeating a week just because these reviews conflict with reviews from different reviewers.

Nyame-Wolf commented on index.html:
```diff
@@ -0,0 +1,49 @@
+                </ul>
+            </nav>
+        </header>
+        <div class="section-header">
```
As highlighted by the previous reviewer, kindly change this tag fro. the general `div` to a `section` tag. It's important to use the correct semantics tags so as not to confuse screen readers.

shurichmond replied:
This is the structure the previous reviewer asked me to follow. You are saying I should remove the div. After another reviewer will come and tell me to change yours again
<section>
  <header>
    <nav>
      <a> your logo </a>
      <ul>
        <li> hamburger icon </li>
      </ul>
    </nav>
  </header>
  <div>
  <h1> here are the tag elements for your name </h1>
  <p> next a `p` for your presentation </p>
  </div>
  <div>
    <a> for your let's connect </a>
    <ul> for your list of social icons
      <li> your list of social icons </li>
    </ul>
  </div>
</section>

shurichmond replied:
<section>
  <header>
    <nav>
      <a> your logo </a>
      <ul>
        <li> hamburger icon </li>
      </ul>
    </nav>
  </header>
  <div>
  <h1> here are the tag elements for your name </h1>
  <p> next a `p` for your presentation </p>
  </div>
  <div>
    <a> for your let's connect </a>
    <ul> for your list of social icons
      <li> your list of social icons </li>
    </ul>
  </div>
</section>

Nyame-Wolf reviewed (changes requested):
# STATUS: REQUIRED CHANGES :recycle: 

Hi there <student> ! , 
![](https://media.giphy.com/media/ReyD4OdRiHeLDCRaEP/giphy.gif)

Great job introducing the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some issues that still need to be addressed for your project to shine :sparkles:.

## Highlights

- Great work using the correct GitHub Flow :100: 
- Linters passing 
- Project Documentation (PR, README) is professional :necktie: 

Congrats on your work so far! :clap:  


 ## Required Changes ♻️

 There are some issues that you still need to work on, but you are nearly there! 
_For more information, check the comments under the review :arrow_down:_


## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

shurichmond committed:
Project resructured and div tag changed to secion

shurichmond commented:
Project restructured according to review 

- [x] Removed this section tag as required by @Nyame-Wolf 
- [x]  Changed general div to a section as required by @Nyame-Wolf

Lizdev-05 reviewed (approved):
Hi @shurichmond, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
<img src="https://media0.giphy.com/media/GCvktC0KFy9l6/giphy.gif"/>
## Highlights
- Correct GitHub Flow 👍

- Good commit history 💰

- Good PR title/description 💯

- All linters passed 🏅

- Personalised README file 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
